javalin/src/main/java/io/javalin/http/staticfiles/JettyResourceHandler.kt

```diff
@@ -18,10 +20,12 @@ import java.io.File
 import javax.servlet.http.HttpServletRequest
 import javax.servlet.http.HttpServletResponse
 
-class JettyResourceHandler : io.javalin.http.staticfiles.ResourceHandler {
+class JettyResourceHandler(val usePrecompressStaticFiles: Boolean = false, compStrat: CompressionStrategy) : io.javalin.http.staticfiles.ResourceHandler {
 
     val handlers = mutableListOf<ResourceHandler>()
 
+    val precompressStaticFiles = PrecompressingResourceHandler(compStrat)
```

Comment:
Hm. Not sure how I feel about this. I don't think we need to pass the compStrat here.